tools/validate_configs/test_configs/new_project.yaml

```diff
@@ -17,6 +17,7 @@
 blueprint_name: new_project
 
 vars:
+  project_id: test_project
```

Comment:
won't this fail validation? since the project is not yet created?

Reply:
Because we do not use a real project ID in make tests, the following validators are disabled during make tests:

- test_project_exists
- test_apis_enabled
- test_region_exists
- test_zone_exists
- test_zone_in_region

Reply:
With project_id set in the module, it fails validation because project_id is set as a deployment variable at the command line (but is unused).